rpc/src/main/kotlin/dk/cachet/carp/rpc/GenerateExampleRequests.kt

```diff
@@ -363,10 +363,14 @@ private val exampleRequests: Map<KFunction<*>, LoggedRequest.Succeeded<*>> = map
     ),
 
     // RecruitmentService
-    RecruitmentService::addParticipant to example(
-        request = RecruitmentServiceRequest.AddParticipant( studyId, participantAccount.emailAddress ),
+    RecruitmentService::addParticipantByEmailAddress to example(
```

Comment:
Given my suggestion to use a method overload here, I was afraid you may run into trouble. But, it should apparently work using:
```
val addParticipantByEmail: suspend (RecruitmentService, UUID, EmailAddress) -> Participant = RecruitmentService::addParticipant
```

Reply:
Currently there is an assumption in `GenerateExampleRequests.generateExampleRequests` that `requestObjectName` is the same as the name of the request but capitalized. So i can't really use overloads in the service, unless we are able to statically specify which request belongs to which service method. I initially also wanted to do overloading, but because of the errors with the example requests I decided not to.

Reply:
> `val addParticipantByEmail: suspend (RecruitmentService, UUID, EmailAddress) -> Participant = RecruitmentService::addParticipant`

It also doesn't work like this: I had to specify the type of the variable to be KSuspendFunction to make the typechecker eat it.